src/net.cpp

```diff
@@ -564,6 +562,22 @@ bool CNode::ReceiveMsgBytes(const char *pch, unsigned int nBytes)
     return true;
 }
 
+unsigned int CNetMessage::FinalizeHeader(CDataStream& s)
+{
+    // Set the size
+    unsigned int nSize = s.size() - CMessageHeader::HEADER_SIZE;
```

Comment:
``s.size()`` returns ``size_t``, so we have ``unsigned int = size_t - int``... is this correct?